packages/framework/src/Resources/config/services.yaml

```diff
@@ -16,7 +16,7 @@ services:
         public: false
 
     Shopsys\FrameworkBundle\:
-        resource: '../../**/*{Calculation,Collector,Dispatcher,Facade,Factory,Generator,Handler,InlineEdit,Listener,Loader,Mapper,Middleware,Parser,Provider,Recalculator,Registry,Repository,Resolver,Service,Scheduler,Subscriber,Transformer}.php'
```

Comment:
What is the benefit of this change?

Reply:
When the Collector classes are autoconfigured, the debug-only services are created in the production env. 

That caused a problem because the ElasticsearchCollector service was passed as one of the dependencies to the service resetter. This class has a dependency on the service `ElasticsearchRequestCollection` and this service is configured only in a dev mode.

Long story short, most collector classes currently present in the shopsys/framework package are debug-only and should not be configured for production env.